contracts/StablecoinConverter.sol

```diff
@@ -121,15 +122,15 @@ contract StablecoinConverter is EpochTokenLocker {
 
     struct TradeData {
         address owner;
-        uint128 volume;
+        uint128 remainingAmount;
         uint16 orderId;
     }
 
     function submitSolution(
         uint32 batchIndex,
         address[] memory owners,  //tradeData is submitted as arrays
         uint16[] memory orderIds,
-        uint128[] memory volumes,
+        uint128[] memory remainingAmounts,
```

Comment:
This rename seems inappropriate. The solution doesn't submit remaining amounts but the executedSellAmounts.